packages/web/components/PageHeader.tsx

```diff
@@ -217,17 +272,46 @@ export const PageHeader: React.FC = () => {
             right="0"
             onClick={onClose}
           />
-          <Box position="absolute" left="calc(50% - 16.5rem)" top="0">
-            <NextImage src={MetaBox} alt="MetaBox" width={528} height={695} />
-          </Box>
+
+          <Box
+            position="absolute"
+            left={`calc(50% - ${content.width} / 2)`}
+            top="0"
+            width={`calc(${content.width} - 0.5rem)`}
+            height={`calc(${content.height} - 3.5rem)`}
+            background={(() => {
+              const opacity = isBackdropFilterSupported() ? 0.75 : 1
+              return (
+                `linear-gradient(
+                  180deg, rgba(76, 63, 143, ${opacity}) 62.76%,
+                  rgba(184, 169, 255, ${opacity}) 100%
+                )`
+              )
+            })()}
+            style={{ backdropFilter: 'blur(10px)' }}
+          />
+          {/* ToDo: switch to BoxedNextImage */}
+          <Image
```

Comment:
BoxedNextImage

Reply:
This one is the only one that's tricky. It has `max-width`  and `max-height` set to `100%`. It will expand beyond it's original dimensions on the y-axis, but not on the x.

I went into the debugger and added random CSS props & tried lots of different values, but it just wouldn't line up when using `BoxedNextImage`.